extensions/python/PythonProcessor.h

```diff
@@ -37,7 +38,7 @@ class PythonProcessor {
 
   void setDescription(const std::string& desc);
 
-  void addProperty(const std::string& name, const std::string& description, const std::string& defaultvalue, bool required, bool el);
+  void addProperty(const std::string& name, const std::string& description, const std::optional<std::string>& defaultvalue, bool required, bool el, const std::optional<int64_t>& validator_value);
```

Comment:
What's a validator value?

Reply:
I added a mapping of the NiFi validators to corresponding MiNiFi Property types and represented them with an int value, with the options listed in the PropertyTypeCode enum. On the python side it's called a validator, but maybe it would be better to change it property_type_code here and there as well.

Reply:
Updated in e8757f9c370b2ec92866a3731af1e6d9ae0edb74